salt/utils/event.py

```diff
@@ -223,17 +226,54 @@ def __load_uri(self, sock_dir, node):
         )
         return puburi, pulluri
 
-    def subscribe(self, tag=None):
+    def subscribe(self, tag):
```

Comment:
This is a backwards incompatible API change, which will break stuff :/ Couldn't this be worked around by assuming a tag of '*'?